forms/src/main/java/org/devgateway/toolkit/forms/wicket/components/table/DataTableAware.java

```diff
@@ -0,0 +1,12 @@
+package org.devgateway.toolkit.forms.wicket.components.table;
+
+import org.apache.wicket.extensions.markup.html.repeater.data.table.DataTable;
+
+/**
+ * @author Nadejda Mandrescu
+ */
+public interface DataTableAware {
```

Comment:
pls add javadoc explaining what this class can helps us with

Reply:
done